README.md

```diff
@@ -3,38 +3,58 @@
 A client library to access the not-so-public Netflix Shakti API.
 
 ## Installation
-```
+```bash
 npm install netflix2
 ```
 
 ## Usage
 All functions take standard Node callbacks:
-```
+```javascript
 function callback (error, result) {}
 ```
 
 ### Initialization
-```
+```javascript
 var Netflix = require('netflix2')
 var netflix = new Netflix()
 ```
 or
-```
+```javascript
 var netflix = require('netflix2')()
 ```
 
 ### Login
 You must call `login` before using any of the other below functions. This will set cookies, API endpoints, and the authURL that must used to make API calls.
-```
+```javascript
 var credentials = {
   email: 'youremail@example.com'
   password: 'yourpassword'
 }
 netflix.login(credentials, callback)
 ```
 
+### Browse
+```javascript
+/**
```

Comment:
Your general idea of explaining the parameters of `netflix.browse` is great, but adding that explenation as JSDoc inside of a README feels strange to me. It would be great if you could move those details outside of the code block.

Reply:
I've added a bit of text above the JSDoc, hope this is ok.